packages/devtools-local-toolbox/package.json

```diff
@@ -20,36 +20,65 @@
     "node": ">=6.9.0"
   },
   "dependencies": {
-    "chrome-remote-debugging-protocol": "^0.0.9",
-    "devtools-config": "^0.0.8",
-    "devtools-modules": "^0.0.8",
-    "devtools-network-request": "^0.0.8",
-    "devtools-sham-modules": "^0.0.8",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "webpack": "1.13.1",
-    "webpack-dev-middleware": "^1.6.1",
-    "webpack-hot-middleware": "^2.12.0",
-    "webpack-env-loader-plugin": "^0.1.4",
-    "extract-text-webpack-plugin": "^1.0.1"
-  },
-  "devDependencies": {
     "amd-loader": "0.0.5",
+    "babel-cli": "^6.7.5",
+    "babel-core": "^6.17.0",
+    "babel-eslint": "^7.1.0",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-module-resolver": "^2.2.0",
+    "babel-plugin-transform-async-to-generator": "^6.16.0",
+    "babel-plugin-transform-es2015-block-scoping": "^6.7.1",
+    "babel-plugin-transform-es2015-destructuring": "^6.6.5",
+    "babel-plugin-transform-es2015-parameters": "^6.7.0",
+    "babel-plugin-transform-es2015-spread": "^6.6.5",
+    "babel-plugin-transform-flow-strip-types": "^6.14.0",
+    "babel-plugin-transform-runtime": "^6.7.5",
+    "babel-plugin-webpack-alias": "^2.1.1",
+    "babel-polyfill": "^6.7.4",
+    "babel-register": "^6.18.0",
     "body-parser": "^1.15.0",
     "check-node-version": "^1.1.2",
+    "chrome-remote-debugging-protocol": "0.0.9",
+    "classnames": "^2.2.5",
     "co": "=4.6.0",
     "css-loader": "^0.25.0",
+    "devtools-config": "^0.0.8",
+    "devtools-modules": "^0.0.8",
+    "devtools-network-request": "^0.0.8",
+    "devtools-sham-modules": "^0.0.8",
+    "eslint-plugin-babel": "^3.3.0",
```

Comment:
Should we add the dependency to `"eslint": "^3.10.0"` here as well? 
I just had a warning about this when doing `npm link` in this package.